editor-extensions/vscode/client/src/extension.ts

```diff
@@ -46,7 +46,7 @@ export function activate(context: ExtensionContext) {
   // Options to control the language client
   let clientOptions: LanguageClientOptions = {
     // Register the server for plain text documents
-    documentSelector: [{ scheme: "file", language: "grain" }],
+    documentSelector: [{ scheme: "file", language: "grain-type" }],
```

Comment:
I think this is not the correct place to change it - this should still be grain otherwise the extension doesn't run against .gr files.

It's line 502 in server.ts where you want to change it to grain-type